### ISSUE TITLE
android fps

### ISSUE THREAD

**Issue by TomWq** [tokkozhin/react-native-neomorph-shadows#32]
A performance problem, I have a big list full of pictures, I use FlatList, and then quickly slide, the list will be very stuck, FPS drop frame is very serious, this problem only appears on Android,ios normal
react-native version:0.62.2

**Comment by tokkozhin** [comment]
Yes, need to be careful when you use a lot of shadow views. Please show the code. it could happen if you used ShadowFlex.

**Comment by AlexBaranovHealthimation** [comment]
I'm using Neomorph (not NeomorphFlex, to avoid re-renders), and general performance on Android is much slower than on iOS, so this issue is definitely real. A screen with 5+ components with shadows renders significantly slower (can lag more than a second on some devices)
